darwin/cli_functions.py

```diff
@@ -899,6 +899,9 @@ def dataset_import(
             dataset_identifier=dataset_slug
         )
 
+        if cpu_limit is not None:
+            use_multi_cpu = True
+
```

Comment:
Without this change, the `--cpu_limt` CLI flag does nothing. This allows multi-threading to work for annotation imports initiated by the CLI